magda-web-client/src/Components/Search/Search.js

```diff
@@ -133,6 +134,34 @@ class Search extends Component {
         });
     }
 
+    /**
+     * counts the number of filters that have active values
+     * this is then appended to the results text on the search page
+     */
+    filterCount = () => {
+        var count = 0;
+        if (this.props.activePublishers.length > 0) {
+            count++;
+        }
+        if (this.props.activeFormats.length > 0) {
+            count++;
+        }
+
+        if (this.props.activeRegion.regionId) {
+            count++;
+        }
+
+        if (this.props.activeDateFrom) {
```

Comment:
What about `activeDateTo` ?

Reply:
@AlexGilleran we don't really need check for both activeDateTo and activeDateFrom. Since once a user selects a date (either "from" or "to"), the other is automatically set as well. Basically every search query is going to have both activeDateTo and activeDateFrom...if that makes sense. Will make the other changes

Reply:
I'll chuck it in anyway